model_compression_toolkit/gptq/pytorch/quantizer/quantization_builder.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+from typing import List, Dict, Tuple
+
+from model_compression_toolkit import GradientPTQConfig
+from model_compression_toolkit.core import common
+from model_compression_toolkit.core.pytorch.constants import KERNEL
+from model_compression_toolkit.exporter.model_wrapper.pytorch.builder.node_to_quantizer import \
+    get_activation_inferable_quantizer_kwargs
+from model_compression_toolkit.gptq.pytorch.quantizer.base_pytorch_gptq_quantizer import \
+    BasePytorchGPTQTrainableQuantizer
+from model_compression_toolkit.quantizers_infrastructure.common.get_quantizer_config import \
+    get_trainable_quantizer_weights_config
+from model_compression_toolkit.qat.pytorch.quantizer.base_pytorch_qat_quantizer import BasePytorchQATTrainableQuantizer
+from model_compression_toolkit.quantizers_infrastructure import QuantizationTarget, BasePyTorchInferableQuantizer
+from model_compression_toolkit.quantizers_infrastructure.common.get_quantizers import get_trainable_quantizer_class, \
+    get_inferable_quantizer_class
+
+
+def quantization_builder(n: common.BaseNode,
+                         gptq_config: GradientPTQConfig,
```

Comment:
Shouldn't with be GradientPTQConfigV2?

Reply:
Not until we remove the old API (I think..)
Unless we don't want to currently support GPTQ with the old API..